assets/pages/oneoff-contributions/helpers/checkoutFormFields.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import { formFieldIsValid } from 'helpers/checkoutForm/checkoutForm';
+import { type PageState as State } from '../oneOffContributionsReducer';
+
+export function formFields(state: State) {
+
+  const fullNameFromState = {
+    value: state.page.user.fullName,
+    ...state.page.checkoutForm.fullName,
+  };
+
+  const emailFromState = {
+    value: state.page.user.email,
+    ...state.page.checkoutForm.email,
+  };
+
+  const fullName = {
+    value: fullNameFromState.value,
+    shouldValidate: state.page.checkoutForm.fullName.shouldValidate,
+    isValid: formFieldIsValid(fullNameFromState),
+  };
+  const email = {
+    value: emailFromState.value,
+    shouldValidate: state.page.checkoutForm.email.shouldValidate,
+    isValid: formFieldIsValid(emailFromState),
+  };
+
+  return { fullName, email };
+}
```

Comment:
This file is a [selector](https://redux.js.org/recipes/computingderiveddata), is it worth calling that out in the file name? So `checkoutSelectors` or something along those lines?

Also, prefer named exports to `default`, so `export { formFields }`.